app/plugin_api.go

```diff
@@ -381,6 +381,10 @@ func (api *PluginAPI) GetProfileImage(userId string) ([]byte, *model.AppError) {
 	return data, err
 }
 
+func (api *PluginAPI) GetEmojiList(page, perPage int) ([]*model.Emoji, *model.AppError){
```

Comment:
Looks like you have a linter error.  This line should be:

```suggestion
func (api *PluginAPI) GetEmojiList(page, perPage int) ([]*model.Emoji, *model.AppError) {
```